UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -352,6 +364,10 @@ public void Unserialize(UndertaleReader reader)
         }
     }
 
+    /// <summary>
+    /// Initialize room by setting <c>ParentRoom</c> property of backgrounds or layers, and calculate room grid size if necessary.
+    /// </summary>
+    /// <param name="calculateGrid">Whether should room grid size be calculated.</param>
```

Comment:
```suggestion
    /// <summary>
    /// Initialize the room by setting <see cref="ParentRoom"/> of Backgrounds or Layers, and optionally calculate the room grid size.
    /// </summary>
    /// <param name="calculateGrid">Whether to calculate the room grid size.</param>
```

Would prefer if Backgrounds and Layers would get a cref as well, its cumbersome from web GH

Reply:
`UndertaleRoom` doesn't have `ParentRoom` property, so it should stay as `<c>`.